### PR TITLE
Adds support for OSC hyperlink sequences.

### LINKED ISSUE #130: Support for OSC hyperlink sequence
I am using `gnome-terminal` to compile a c++ program, a typical output for a warning can look like this:

gcc1

The blue text `-Warray-bounds` is a hyperlink coded as an [OSC sequence](https://en.wikipedia.org/wiki/ANSI_escape_code#OSC_(Operating_System_Command)_sequences). When I run `ansi2html` on this output I get something like the following:
gcc2
which shows (notice the garbled output in pink) that the OSC hyperlink sequence is probably not supported by the program.

### PR DESCRIPTION
**Pull request by hakonhagland**
Fixes #130 : adds support for OSC hyperlink sequences.

### CODE REVIEW

File: ansi2html/converter.py [hunk 1]
```diff
@@ -347,9 +373,36 @@ def _vt100_box_drawing():
...
+            yield ansi[last_end:]
 
         state = _State()
-        inside_span = False
+        for part in _osc_link(ansi):
+            if isinstance(part, OSC_Link):
+                yield part
+            else:
+                if ((sys.version_info.major == 3 and sys.version_info.minor >= 3)
+                        or sys.version_info.major > 3):
```
Comment (ssbarnea) [inline ansi2html/converter.py:393]: This makes no sense because we already require py36+, please remove variant for relic-pythons.

Comment (hakonhagland) [reply ansi2html/converter.py]: Thanks for the comment, I have removed the check in a new commit.

**Review by ssbarnea — APPROVED** [review]

**Comment by ssbarnea** [comment]
@hakonhagland Linting is failing. I also sent you an invite https://github.com/pycontribs/ansi2html/invitations so you will not have to wait for me to approve CI executions. 

PS. I could really use some help with #135

**Comment by hakonhagland** [comment]
> Linting is failing.

@ssbarnea I added a new commit to fix this.

> I also sent you an invite https://github.com/pycontribs/ansi2html/invitations so you will not have to wait for me to approve CI executions.

Thanks! I have accepted the invite. I am quite new to Python, so I will use this as a learning experience.

> PS. I could really use some help with #135

I will try to help but I am not familiar with all the testing and GitHub actions yet.. Is there anything in particular I should look at?

**Comment by hakonhagland** [comment]
Any idea why the linter is failing now? 
The [log](https://github.com/pycontribs/ansi2html/pull/131/checks?check_run_id=3487356580)  file does not give much:

```
ERROR: InvocationError for command /home/runner/work/ansi2html/ansi2html/.tox/lint/bin/python -m pre_commit run --all (exited with code 1)
```

**Comment by ssbarnea** [comment]
@hakonhagland Yes. Just run "tox -e lint" locally and include the modified files in your change. black is reformatting the code so you do not have to think about how to follow the style guide too much.

**Pull request merged**